packages/completer/src/reconciliator.ts

```diff
@@ -50,15 +50,19 @@ export class ProviderReconciliator implements IProviderReconciliator {
 
   fetchInline(
     request: CompletionHandler.IRequest,
-    trigger: InlineCompletionTriggerKind
+    trigger: InlineCompletionTriggerKind,
+    isMiddleOfLine: boolean
```

Comment:
To match the public API definition:

```suggestion
    isMiddleOfLine?: boolean
```